MEK_Installer.pyw

```diff
@@ -437,6 +440,18 @@ class MekInstaller(tk.Tk):
         self.tk.call("tk", "scaling", "1.666")
         self.minsize(480, 300)
 
+        if ((sys.version_info[0] == PYTHON_MIN_MAJOR_VER and sys.version_info[1] < PYTHON_MIN_MINOR_VER) or
+            sys.version_info[0] < PYTHON_MIN_MAJOR_VER):
+            messagebox.showinfo(
+            "Your version of Python is too old",
+            "The minimum required version of Python for the MEK is %d.%d.\n"
+            "You have version %d.%d, which is behind that by %d major versions and %d minor versions.\n\n"
```

Comment:
This `by %d major versions and %d minor versions` is not useful information because it doesn't change what the user needs to do. I think it just adds visual clutter and at worst confuses the user if they don't know what it means.

Also, I feel starting with "your Python is too old" doesn't really provide the right context. For unfamiliar users, they might ask: "What's Python?", "So what?", and "Is it mandatory?". The next line gives more context, but it would be better to lead with that.

For these lines I'd suggest:

```
MEK requires a minimum Python version of <min_ver>.
Your system has Python version <sys_version>.
...
```

Reply:
The first line is actually the title bar. Them

Reply:
Title could be "Python upgrade required". The title could be easily missed, so it's good to re-iterate that line.